### ISSUE TITLE
Integrate GRPC

### ISSUE THREAD

**Issue by ronaldpetty** [cncf/demo#191]
Need to update demo app to include GRPC usage.

**Comment by namliz** [comment]
@ronaldpetty agreed, if you have a none contrived example in mind I'm very open to suggestions.
